backend/dashboard/views/pages/design-system/design-system-content/ChelListsTables.vue

```diff
@@ -1,9 +1,9 @@
 <template lang="pug">
 content-outlet(title='Lists & Tables')
   .content-unit
-    h4.unit-name Lists
+    h4.unit-name TBD...
 
-    .unit-description list UI elements used in the app
+    .unit-description TBD...
```

Comment:
Are these not implemented yet? I'm curious because there are the `TableTemplate.vue` and `TableRow.vue` files...

Reply:
@taoeffect They are implemented and used in various dashboard pages:

<img src='https://github.com/user-attachments/assets/3b88b653-125b-4c7d-b8b9-2be01678a614' width='520'>

<br><br>

![image](https://github.com/user-attachments/assets/d995e5e3-2376-4a7a-ae84-1ce91306754c)


`TBD..` here means I have not added demos for these in this design-system section.
Also, FYI `TableTemplate.vue` and `TableRow.vue` are components used for design-system demo. Not UIs in the the dashboard pages.